README.md

```diff
@@ -3,93 +3,100 @@
-- Support for distributed training using FSDP from PyTorch Distributed
-- YAML configs for easily configuring training runs
-- [Upcoming] Support for lower precision dtypes and quantization techniques from [TorchAO](https://github.com/pytorch-labs/ao)
-- [Upcoming] Interop with various inference engines
+| Model                                         | Sizes     |
+|-----------------------------------------------|-----------|
+| [Llama2](https://llama.meta.com/llama2/)   | 7B, 13B [[models](torchtune/models/llama2/_model_builders.py), [configs](recipes/configs/llama2/)]        |
+| [Mistral](https://huggingface.co/mistralai)   | 7B [[model](torchtune/models/mistral/_model_builders.py), [configs](recipes/configs/mistral/)] |
+| [Gemma](https://huggingface.co/collections/google/gemma-release-65d5efbccdbb8c4202ec078b)   | 2B [[model](torchtune/models/gemma/_model_builders.py), [configs](recipes/configs/gemma/)] |
+
+We'll be adding a number of new models in the coming weeks, including support for 70B versions and MoEs.
 
 &nbsp;
 
-The library currently supports the following models and fine-tuning methods.
+### Fine-tuning recipes
 
-| Model                                         | Sizes     |   Finetuning Methods |
-|-----------------------------------------------|-----------|-----------------------------------------------------------|
-| [Llama2](torchtune/models/llama2/_model_builders.py)   | 7B        | Full Finetuning [[single device low memory](recipes/configs/llama2/7B_full_low_memory.yaml),  [distributed](recipes/configs/llama2/7B_full.yaml)] LoRA [[single device](recipes/configs/llama2/7B_lora_single_device.yaml),  [distributed](recipes/configs/llama2/7B_lora.yaml)] QLoRA [[single device](recipes/configs/llama2/7B_qlora_single_device.yaml)] |
-| [Llama2](torchtune/models/llama2/_model_builders.py)   | 13B       | [Full Finetuning](recipes/configs/llama2/13B_full.yaml), [LoRA](recipes/configs/llama2/13B_lora.yaml)
-| [Mistral](torchtune/models/mistral/_model_builders.py)   | 7B       | Full Finetuning [[single device low memory](recipes/configs/mistral/7B_full_low_memory.yaml), [distributed](recipes/configs/mistral/7B_full.yaml)] LoRA [[single device](recipes/configs/mistral/7B_lora_single_device.yaml), [distributed](recipes/configs/mistral/7B_lora.yaml)] QLoRA [[single device](recipes/configs/mistral/7B_qlora_single_device.yaml)]
-| [Gemma](torchtune/models/gemma/_model_builders.py)   | 2B        | [Full Finetuning](recipes/configs/gemma/2B_full.yaml)
+torchtune provides the following fine-tuning recipes.
 
+| Training                           | Fine-tuning Method                 |
+|------------------------------------|------------------------------------|
+| Distributed Training [1 to 8 GPUs] | Full [[code](recipes/full_finetune_distributed.py), [example](recipes/configs/llama2/13B_full.yaml)], LoRA [[code](recipes/lora_finetune_distributed.py), [example](recipes/configs/llama2/13B_lora.yaml)] |
```

Comment:
Should we put 2 to 8 for distributed training?

Reply:
They do work on 1 GPU, just not with all of the memory efficiency (eg: bitsandbytes)?

Reply:
Ah OK I thought at some point we had made the assumption that they didn't. But iirc that was for meta device stuff, which we since unwound.